sshenc.sh

```diff
@@ -75,6 +103,9 @@ elif [[ -e "$private_key" ]]; then
     stdin=`cat`
     keys_enc=$(echo "$stdin" | awk '/-- keys/{f=1;next} /-- \/keys/{f=0} f')
     cypher=$(echo "$stdin" | sed -e '1,/-- \/keys/d')
+    install -m 0600 "$private_key" "$temp_dir/private_key"
+    ssh-keygen -p -m PEM -N '' -f "$temp_dir/private_key"
+
```

Comment:
there is a potential security issue here, as i make a password-less copy of the ssh private key in a temporary directory.

I don't know if the script supported private ssh keys protected by password before. now it does :)